packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Combination/BasicCombinationGraph/BarGraphWithLine.2.doc.mdx

```diff
@@ -0,0 +1,38 @@
+import CombinationGraph from '../../../example/Graphs/CombinationGraph/BasicCombinationGraphs/BarGraphWithLine?dev-site-example';
+import PropsTable from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault.js?dev-site-codeblock';
+import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/BarGraphWithLineData.js?dev-site-codeblock';
+require('details-polyfill')
+
+# Bar Graph With Line
+This is a Simple Bar Graph Combined with Line.
```

Comment:
Is "Combined" used here as part of the name of the graph or a verb? If this should be updated, please also update this for the other docs below.

```suggestion
This is a Simple Bar Graph combined with Line.
```

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/150/commits/768d7923e4f325eaef40c8caa6ff6753069fc96e#diff-c5f198d3ceba8c26d769396dbfc28b4ab83a9c7f4889987bc628b9193d134ed4R8).

Reply:
also nit: lowercase s

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/150/commits/7f78b0c5f428f721ba276d4c8c689a0e2c5b56fe#diff-c5f198d3ceba8c26d769396dbfc28b4ab83a9c7f4889987bc628b9193d134ed4R8).